src/analytics.ts

```diff
@@ -353,3 +353,17 @@ export const getCustomerHash = (): Promise<{hash: string}> =>
     postMessageToNativeApp({
         type: 'GET_CUSTOMER_HASH',
     });
+
+export const setTrackingProperty = (
+    system: string,
```

Comment:
we could type this as a string union with the allowed systems:
```suggestion
    system: 'palitagem' | 'medalia',
```